src/signals/incident-management/containers/IncidentDetail/components/History/index.js

```diff
@@ -1,27 +1,50 @@
 import React from 'react';
-import { historyType } from 'shared/types';
 
+import styled from '@datapunt/asc-core'
+import { themeColor, Heading, styles, Column } from '@datapunt/asc-ui'
 import { string2date, string2time } from 'shared/services/string-parser/string-parser';
+import { historyType } from 'shared/types';
+
+const StyledH4 = styled(Heading)`
+  ${styles.HeaderStyles} {
+    margin-top: 1.33em;
+    margin-bottom: 1.33em;
+  }
+`;
+
+const StyledHistoryListItem = styled(Column)`
+  margin-bottom: 16px;
```

Comment:
you should use ${themeSpacing(1)} there for 4 pixel spacing: if the values are not correct then adjust to closest.

Reply:
we also use px everywhere

Reply:
Aangepast